src/main/java/hana/command/EventCommand.java

```diff
@@ -0,0 +1,60 @@
+package hana.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import hana.HanaException;
+import hana.storage.Storage;
+import hana.task.Event;
+import hana.task.Task;
+import hana.tasklist.TaskList;
+import hana.ui.Ui;
+
+/**
+ * Represents a command to add Event task.
+ * When executed, this command will add new Event task, print a message and save.
+ */
+public class EventCommand extends Command {
+    private String description;
+    private LocalDateTime from;
+    private LocalDateTime to;
+
+    /**
+     * Constructs a new EventCommand with a description, start and end date.
+     *
+     * @param input The input from user.
+     * @throws HanaException If an error occurs.
+     */
+    public EventCommand(String input) throws HanaException {
+        try {
+            String[] parts = input.substring(5).split(" /from | /to ");
+            if (parts.length < 3) {
+                throw new HanaException("Event task must have a description, start time, and end time.");
+            }
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+            this.from = LocalDateTime.parse(parts[1].trim(), formatter);
+            this.to = LocalDateTime.parse(parts[2].trim(), formatter);
```

Comment:
can consider to have 2 constants : INPUT_FORMATTER and OUTPUT_FORMATTER to format the input date and times